### PR TITLE
update link to new jira dooks

### PR DESCRIPTION
**Pull request by svelle**
saves a click and avoids more confusion as this page is the first one that comes up on google.

### CODE REVIEW

File: source/integrations/jira.rst [hunk 1]
```diff
@@ -3,6 +3,6 @@
 Jira Plugin 
 ================================
 
-Documentation has moved to `Jira plugin repository <https://github.com/mattermost/mattermost-plugin-jira>`_.
+The Jira/Mattermost plugin documentation is currently being updated and relocated to a new location: https://mattermost.gitbook.io/jira-plugin/
```
Comment (amyblais) [inline source/integrations/jira.rst]:
```suggestion
The Jira/Mattermost plugin documentation is currently being updated and relocated to a new location: https://mattermost.gitbook.io/jira-plugin/.
```

File: source/integrations/jira.rst [hunk 2]
```diff
@@ -3,6 +3,6 @@
...
 
 Learn more about how to integrate with your other `Atlassian tools <https://docs.mattermost.com/deployment/atlassian-integrations.html>`.
```
Comment (amyblais) [inline source/integrations/jira.rst]: Not part of this PR but the link formatting here is not correct.

Comment (svelle) [reply source/integrations/jira.rst]: woudl this be better?
